src/app/pages/main/main.module.ts

```diff
@@ -5,12 +5,20 @@ import { NgModule } from '@angular/core';
 import { SharedModule } from '@shared/shared.module';
 import { SidenavComponent } from './components/sidenav/sidenav.component';
 import { CommunityPreviewComponent } from './components/community-preview/community-preview.component';
+import { CommunityEditorComponent } from './components/community-editor/community-editor.component';
+import { ClipboardModule } from '@angular/cdk/clipboard';
 
-const COMPONENTS = [MainComponent, MapComponent, SidenavComponent, CommunityPreviewComponent];
+const COMPONENTS = [
+  MainComponent,
+  MapComponent,
+  SidenavComponent,
+  CommunityPreviewComponent,
+  CommunityEditorComponent,
+];
 
 @NgModule({
   declarations: COMPONENTS,
-  imports: [SharedModule, GoogleMapsModule],
+  imports: [SharedModule, GoogleMapsModule, ClipboardModule],
```

Comment:
```suggestion
  imports: [SharedModule, GoogleMapsModule],
```

ClipboardModule is already added in `SharedModule` :+1: